integrations/google_ai/tests/generators/chat/test_chat_gemini.py

```diff
@@ -207,22 +207,35 @@ def get_current_weather(location: str, unit: str = "celsius"):  # noqa: ARG001
     get_current_weather_func = FunctionDeclaration.from_function(
         get_current_weather,
         descriptions={
-            "location": "The city and state, e.g. San Francisco, CA",
+            "location": "The city and state, e.g. San Francisco",
```

Comment:
Why change this? 🤔

Reply:
The model generated different responses for this., e.g. it was sometimes `Berlin, Germany` and sometimes `Berlin, DE`. It was making assertion uncertain.

Reply:
Oh ok, I would edit the description to reflect that change then. 

```suggestion
            "location": "The city, e.g. San Francisco",
```

Reply:
Same for the one below obviously.